hackathons/virtual_2.0.html

```diff
@@ -0,0 +1,1034 @@
+    <div class="container">
+
+      <div class="row">
+         <div class="col s12">
+            <div class="row">
+              <div class="col l11 s12 offset-l1" style="color: #fff">
+                  <h3 style="font-family: 'Oswald', sans-serif; color: #62a337;">What?</h3>
+                  <p>DefHacks Virtual 2020 hackathon (What is a hackathon?) is a free 36-hour event for high school and college students. It is a worldwide event in which students of all skill levels around can collaborate to create innovative projects to solve the worlds problems. Students can also participate in workshops to learn new skills, ask questions to top tech minds, participate in fun mini-events, and win amazing prizes!</p>
+              </div>
+            </div>
+            <div class="row">
+            <div class="col l3 s12 offset-l1" style="color: #fff">
+                <h3 style="font-family: 'Oswald', sans-serif; color: #62a337;">Who?</h3>
+                <p>Any high school or college student across the world is welcome to apply! However, we might not be shipping swag to students located outside of the United States.</p>
+            </div>
+            <div class="col l3 s12 offset-l1" style="color: #fff">
+                <h3 style="font-family: 'Oswald', sans-serif; color: #62a337;">When?</h3>
+                <p>Saturday, November 14, 2020 12:00 PM PST - Sunday, November 15, 2020 11:59 PM PST</p>
+            </div>
+              <div class="col l3 s12 offset-l1" style="color: #fff">
+                <h3 style="font-family: 'Oswald', sans-serif; color: #62a337;">Where?</h3>
+                <p>The hackathon will be COMPLETELY online. We'll conduct the majority of the hackathon on slack, and the opening/closing sessions will be on a YouTube livestream!</p>
+              </div>
+            </div>
+         </div>
+       </div>
+
+       <style>
+
+       .question h4 {
+         font-family: 'Oswald', sans-serif;
+       }
+       </style>
+
+
+       <div class="row">
+         <div class="col 12">
+
+         </div>
+       </div>
+
+
+    </div>
+  </div>
+  <div class="seperator-top hide-on-small-only" style="margin-top: -50px; margin-bottom: -1px;border-bottom: 51px solid #62a337; border-right: 1500px solid #303030; overflow: hidden; -webkit-transform: scaleX(1); transform: scaleX(1);"></div>
+
+  <div style="width: 100%; background-color: #62a337; height: 30px; margin-top: -10px;"></div>
+
+
+  <div class="section" style="width: 100%; background-color: #fff; padding: 40px 0px">
+    <div class="container">
+      <div class="row">
+         <div class="col s12">
+           <h2 style="color: #303030; margin-bottom: 18px; margin-top: 30px; text-align: center;"><i class="fas fa-user-plus" aria-hidden="true"></i>Interested in attending?</h2>
+           <h3 align="center">
+           	Subscribe for our Newsletter to know the further Announcements.
```

Comment:
Change text to "Registrations aren't open yet - but subscribe to our newsletter, and we'll let you know when they do!

This needs to be a pargaraph tag not h3 tag